sphinx/quickstart.py

```diff
@@ -266,6 +268,58 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = '%(project_fn)sdoc'
 
+# -- Options for Apple Help output ----------------------------------------
```

Comment:
We haven't added potential options for other "minor" builders to the quickstart config, such as devhelp or qthelp, to keep the length from exploding. Since your config values have reasonable defaults, I would prefer that here too.